pandas/tests/window/test_rolling.py

```diff
@@ -81,8 +90,77 @@ def test_invalid_constructor(frame_or_series, w):
         c(window=2, min_periods=1, center=w)
 
 
+@pytest.mark.parametrize(
+    "window",
+    [
+        timedelta(days=3),
+        Timedelta(days=3),
+        "3D",
+        ExpandingIndexer(window_size=3),
```

Comment:
`ExpandingIndexer`, `ExponentialMovingWindowIndexer`, and `GroupbyIndexer` can be removed since they are tested below